src/action/Button/Button.tsx

```diff
@@ -139,19 +158,71 @@ const buttonKind = (kind: Kind = "normal", variant: Variant = "primary") => {
       return css`
         background-color: transparent;
         color: ${color};
-        border: 0;
+        border: 2px solid transparent;
 
         &:hover:enabled {
           opacity: 0.8;
         }
       `;
+    case "text":
+      return css`
+        background-color: transparent;
+        color: ${color};
+        border: 2px solid transparent;
+
+        &:hover {
+          background-color: ${colorLight};
+        }
+      `;
+  }
+};
+
+const blockSize = (block?: boolean) => {
+  return css`
+    width: ${block ? "100%" : "auto"};
+  `;
+};
+
+export type Align = "left" | "center" | "right";
+const alignContent = (align: Align = "center") => {
```

Comment:
there isn't a **default** in your switch, I'm wondering if it could be clearer to specify **center** there, especially since **center** isn"t declaring any css property.
Not sure about this comment though

Reply:
We need to talk about it

Reply:
We don't change it